charts/datadog/values.yaml

```diff
@@ -77,6 +77,13 @@ datadog:
     # datadog.kubeStateMetricsNetworkPolicy.create -- If true, create a NetworkPolicy for kube state metrics
     create: false
 
+  kubeStateMetricsCore:
+    # datadog.kubeStateMetricsCore.enabled -- Enable the kubernetes_state_core check in the Cluster Agent
+    enabled: false
+
+    # datadog.kubeStateMetricsCore.ignoreLegacyKSMCheck -- Disable the auto-configuration of lagacy kubernetes_state check (taken into account only when datadog.kubeStateMetricsCore.enabled is true)
+    ignoreLegacyKSMCheck: true
```

Comment:
Is it necessary to offer this? Setting `datadog.kubeStateMetricsCore.enabled` to false if, for some reason, core does not work well, should be enough no?

Reply:
`ignoreLegacyKSMCheck` is only taken into consideration if `kubeStateMetricsCore.enabled=true`. so yes disabling `kubeStateMetricsCore` is enough and guaranteed already.
`ignoreLegacyKSMCheck` can be useful though during the migration phase, users might be interested in enabling both checks at first